### DISCUSSION TITLE
DataGrid Footer with RowCount on filtered Datagrid

### DISCUSSION THREAD

**Discussion by thg87** [blazority/support#90] [Q&A] [answered]
Hello everybody,

i have a problem with the datagrid. I want display the current Row Count in die DataGrid Footer. This is no Problem so far.
But when i filter the items by one column, my row count in the datagrid footer didnt change. 
Have anybody an idea, how to solve my problem?

**Comment by karan-kang** [comment]
@thg87 Would it be possible to share a snippet of your template?

**Reply by thg87** [reply]
< Datagrid TRowData="DataObject" Items="Result"  Placeholder="Keine Treffer gefunden.">      

        <DatagridColumn Title="Id" TRowData="DataObject" Field="(c) => c.MandantId" 
            CanSort="true" CanFilter="true" FilterImmediately="false" FilterMatch="(f) => StartsWithMatch(f)" />

        <DatagridFooter TRowData="DataObject">

            <DatagridFooterDescription TRowData="DataObject">
                @Result.Count Datensätze
            </DatagridFooterDescription>

        </DatagridFooter>

    </Datagrid>
}

@code {

    public List<DataObject> Result { get; set; } = new List<DataObject>();
}

**Comment by thg87** [comment, marked as answer]
I have found the property "ViewData" on the DatagridInstance. It solve my problem.
